pprint/test/src-2/test/pprint/TPrintTests.scala

```diff
@@ -235,6 +235,22 @@ object TPrintTests extends TestSuite{
       import scala.reflect.runtime.universe._
       check[Nothing]("Nothing")
     }
+    test("polytype"){
+      import scala.language.implicitConversions
+      type Foo[+A] = Unit
+      trait Bar[F[_], A]
+      object Bar {
+        implicit def anyToBar[A](a: A): Bar[Foo, A] = new Bar[Foo, A] {}
+        def apply[F[_], A](b: Bar[F, A]): Bar[F, A] = b
+      }
+      object Print {
+        def tprint[A](a: A)(implicit t: pprint.TPrint[A]) = t.render
+      }
+      val rendered = Print.tprint(Bar(1)).toString()
+
+      val is213Plus = classOf[Seq[Int]].getName != "scala.collection.Seq"
```

Comment:
You can use `util.Properties.versionNumberString`

Reply:
I stole this from earlier in the file which notes I can't on ScalaJS apparently

https://github.com/com-lihaoyi/PPrint/blob/cb8229fd6b0117bafc2d0c9b9b79f0a6563b6baa/pprint/test/src-2/test/pprint/TPrintTests.scala#L61-L62

Reply:
TIL